leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -171,13 +179,14 @@ var ElementInspector = Backbone.View.extend({
     /**
      * Initializes the element inspector using previously defined templates
      */
-    render: function(cell) {
-
-        this.cell = cell; // Save the clicked node's backbone model
+    
+    render: function() {
 
         // Save the Intention object from the global model variable to
-        // this.intention
-        this.intention = model.getIntentionByID(this.cell.attributes.nodeID);
+        //this.model.intention
+        console.log(this.model);
+        console.log(this.model.intention);
+        this.model.intention = model.getIntentionByID(this.model.attributes.nodeID);
 
         this.$el.html(_.template(this.template)());
```

Comment:
```suggestion
        this.$el.html(_.template($(this.template).html())(this.model.toJSON()));
```
This should inject your model parameters directly into the template